lisa/operating_system.py

```diff
@@ -244,6 +232,18 @@ def _install_packages(self, packages: Union[List[str]]) -> None:
         )
 
 
+class Redhat(Fedora):
+    @classmethod
+    def name_pattern(cls) -> Pattern[str]:
+        return re.compile("^rhel|Red|Scientific|acronis|Actifio$")
+
+    def _initialize_package_installation(self) -> None:
+        self._node.execute("yum -y update", sudo=True, timeout=1200)
+
+    def _install_packages(self, packages: Union[List[str]]) -> None:
+        self._node.execute(f"yum install -y {' '.join(packages)}", sudo=True)
+
+
```

Comment:
What's being reused from Fedora?

Reply:
The class type can be used in `isinstance`, and it can simplify logic in tools. For example, if fedora and redhat use same package name, or other same behavior.

Reply:
So you're speculating for future code, is that it? Because I don't see tools related to those distro classes, am I missing something?

Reply:
`Fedora` is not an endorsed distro, so we don't have good coverage on it. Below is an example, how the class is used to detect distro. If Federa has same package name as RedHat, it can be changed to Fedora easily to make `lsvmbus` supports Fedora.

LISAv3 is at initial stage, there are 400 v2 test cases will be migrated. In v2, there are a lot of code to handle different distros in many levels, I hope to centralize and decouple those logics. At least, not to write it in test case.

https://github.com/microsoft/lisa/blob/002acab91b5af5c46133a5a8d601c373601ef529/lisa/tools/lsvmbus.py#L149

Reply:
Oh, I see, so they get the same package names. We could have a method retrieving package names for distros later on, having a base class dict hold that (we agree on common keys for each entry).

Reply:
We can see if a package name dict helps or not later. In some cases, the package name is not enough, it needs extra logic. There is an example in `lsvmbus.py` for Ubuntu also. Anyway, we will learn more after more test cases onboarded.